README.md

```diff
@@ -339,15 +339,13 @@ auto e = Eventual<T>()
         });
     thread.detach();
   })
-  | [](auto&& result) {
+  | Then([](auto&& result) {
     // Return some value ***synchronously**.
     return stringify(result);
-  };
+  });
 ```
 
-### `Then`
-
-When your continuation is ***asynchronous*** (i.e., you need to create another eventual based on the result of an eventual) but you *don't* need the explicit control that you have with an `Eventual` you can use `Then`:
+In many cases you can be even more implicit and just use a C++ lambda directly too:
```

Comment:
Is it still possible?

Reply:
As Nikita points out, let's remove this section, since it's not currently correct.